server/src/test/java/io/crate/statistics/StatsUtils.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to Crate.io GmbH ("Crate") under one or more contributor
+ * license agreements.  See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.  Crate licenses
+ * this file to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial agreement.
+ */
+
+package io.crate.statistics;
+
+import java.util.Collection;
+
+import io.crate.types.DataType;
+
+public final class StatsUtils {
+
+    public static <T> ColumnStats<T> statsFromValues(DataType<T> dataType, Collection<T> values) {
+        ColumnSketchBuilder<T> builder = dataType.columnStatsSupport().sketchBuilder();
+        builder.addAll(values);
+        return builder.toStats();
+    }
+
+    public static <T> ColumnStats<T> statsFromValues(DataType<T> dataType, Collection<T> values, int nullCount) {
+        ColumnSketchBuilder<T> builder = dataType.columnStatsSupport().sketchBuilder();
+        builder.addAll(values);
+        for (int i = 0; i < nullCount; i++) {
+            builder.add(null);
+        }
+        return builder.toStats();
+    }
```

Comment:
How about a `toStats(values)` on `ColumnStatsSupport` ? It would avoid another indirection and I think help a bit with the discoverability.

Reply:
Oh I missed that this is in the test sources. Agree that in that case it's better to keep as is.